pv_site_api/main.py

```diff
@@ -355,6 +357,26 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     return res
 
 
+@app.get("/enode/link")
+def get_enode_link(redirect_uri: str, session: Session = Depends(get_session)):
+    """
+    ### Returns a URL from Enode that starts a user's Enode link flow.
+    """
+    if int(os.environ["FAKE"]):
+        return make_fake_enode_link()
+
+    client = session.query(ClientSQL).first()
+    assert client is not None
+
+    with httpx.Client() as httpx_client:
+        data = {"vendorType": "inverter", "redirectUri": redirect_uri}
+        res = httpx_client.post(
+            f"https://enode-api.production.enode.io/users/{client.client_uuid}/link", data=data
+        ).json()
+
+    return res["linkUrl"]
```

Comment:
Do you think we can just do this with a [fastapi RedirectResponse?](https://fastapi.tiangolo.com/es/advanced/custom-response/#redirectresponse)

Reply:
nevermind

Reply:
Opted for this in the end. On the frontend, we can just use a `fetch` as usual, we just won't read the response body (Enode Link UI) and instead just history push the page to the `Response.url`.